docs/source/optim.rst

```diff
@@ -303,6 +314,76 @@ algorithms.
     lr_scheduler.OneCycleLR
     lr_scheduler.CosineAnnealingWarmRestarts
 
+How to utilize named parameters to load optimizer state dict
+--------------------------------------------
+
+The function :func:`~Optimizer.load_state_dict` ignores the optional ``param_names`` input in the
+loaded state dict if present (to maintain compatibility).
```

Comment:
Is it better to say it treats like any other unfamiliar key in the state and will populate by just copy/paste?